### ISSUE TITLE
An error was encountered while requesting a list of devices. Response was [{"error":402}]

### ISSUE THREAD

**Issue by ebreen** [bwp91/homebridge-ewelink#20]
Can someone help me, when i start homebridge it fails to request list of devices.

[2018-4-23 16:11:52] [eWeLink] A total of [0] accessories were loaded from the local cache
[2018-4-23 16:11:52] [eWeLink] Requesting a list of devices from eWeLink HTTPS API at [https://eu-api.coolkit.cc:8080]
[2018-4-23 16:11:52] Homebridge is running on port 51826.
[2018-4-23 16:11:53] [eWeLink] An error was encountered while requesting a list of devices. Response was [{"error":402}]

this is my config

{
    "bridge": {
        "name": "Homebridge",
        "username": "CC:22:3D:E3:CE:30",
        "port": 51826,
        "pin": "111-11-111"
    },
    
    "description": "boom",

    "accessories": [
        
    ],

    "platforms": [
        {
        "platform" : "eWeLink",
        "name" : "eWeLink",
        "authenticationToken" : "xxxxxxxx",
        "apiHost" : "eu-api.coolkit.cc:8080",
        "webSocketApi" : "eu-long.coolkit.cc"
        }
    ]
}

**Comment by Stuart-Richards** [comment]
Replace eu-long.coolkit.cc with eu-pconnect3.coolkit.cc

**Comment by Stuart-Richards** [comment]
check your authentication token is correct? It should look something like this

 "authenticationToken": "ec617d819b4ee12b41305f1286bfa27918474c07",

**Comment by ebreen** [comment]
Yeah i have the right auth token, i just swapped it out with xxxx when posting here :)

**Comment by ebreen** [comment]
What could error 402 mean

**Comment by jwhits** [comment]
@ebreen I had the 402 error, using Charles I was also seeing the 402 error when opening the ewelink app. My token in Charles was the same as what I had in homebridge. I tried logging out of the app and logging back in, a new token was generated. No 402 in Charles, updated hombridge config with new token. Got past the 402 then got the new error ENOTFOUND so I updated the homebridge config to `"webSocketApi" : "us-pconnect3.coolkit.cc"`
 Mine is all working again
